interface/weno/validate.php

```diff
@@ -71,8 +71,8 @@
     exit;
 }
 $npiLength = strlen($pharmacy['npi']);
-if (empty($pharmacy['npi'] || $npiLength < 10)) {
-    print xlt("Pharmacy missing NPI  or less than 10 digits"). "<br>";
+if (empty($pharmacy['npi'] || $npiLength < 7)) {
```

Comment:
NPI should remain at 10 characters however the NCPDP is always 7 characters so,
if (empty($pharmacy['ncpdp']) || $ncpdpLength < 7) {`
Is correct fix.